Analysis/include/SampleDescriptor.h

```diff
@@ -20,11 +20,14 @@ struct AnalyzerSetup {
     Period period;
     SignalMode mode;
     QCDmethod qcd_method;
+    EventRegionSet qcd_shape;
     DiscriminatorWP tauID_wp;
     std::vector<std::string> final_variables;
     std::vector<double> pt_sel_bins;
-    bool apply_mass_cut{false}, apply_os_cut{true}, apply_iso_cut{true};
+    bool apply_mass_cut{false}, apply_os_cut{true}, apply_tau_iso{true};
```

Comment:
where "apply_tau_iso" is used? I don't understand why it is needed at the ana config.